### ISSUE TITLE
The `invalid.deprecated.entity.other.attribute-name.html` warning for the `align` attribute in `Vue` is confusing

### ISSUE THREAD

**Issue by nandordudas** [subframe7536/vscode-theme-maple#2]
> [!NOTE]
>
> - `Windows 11 23H2` with `WSL 2`
> - VSCode: `1.93.1`
> - `Maple-Mono 6.4`
> - Extension: `v0.4.6`

## Description

The `align` attribute appears with strikethrough text, which is unexpected for custom elements.

## Example

A: Without settings change

![image](https://github.com/user-attachments/assets/c31cf2cf-d8a4-482e-8af3-465a7e560b91)

B: With settings change

```jsonc
{
  "editor.tokenColorCustomizations": {
    "[Maple Light][Maple Dark]": {
      "textMateRules": [
        {
          "scope": [
            "source.vue invalid.deprecated.entity.other.attribute-name.html"
          ],
          "settings": {
            "fontStyle": ""
          }
        }
      ]
    }
  }
}
```

![image](https://github.com/user-attachments/assets/1490067a-d4d7-4d78-bc28-cbb53ac4f2ef)

**Comment by nandordudas** [comment]
I'm not sure if this is a bug, an improvement, or something that can be resolved with local settings.

**Comment by subframe7536** [comment]
Fixed in v0.4.7
